tce/src/applibs/Scheduler/ProgramRepresentations/CFG/BasicBlockNode.hh

```diff
@@ -102,6 +102,16 @@ public:
     void setLoopScheduled() { loopScheduled_ = true; }
     void setBBOwnership(bool ownership = true) { bbOwned_ = ownership; }
 
+    // hwloop flags
+    bool
+    isHWLoop() const {
+        return hwloop_;
+    }
+    void
+    setHWLoop() {
```

Comment:
add a bool parameter so it's possible to unset?

Reply:
Done